caffe2/python/pybind_state.cc

```diff
@@ -610,6 +612,81 @@ void addObjectMethods(py::module& m) {
       .def_property_readonly("description", &OpSchema::Argument::description)
       .def_property_readonly("required", &OpSchema::Argument::is_required);
 
+  py::class_<onnx_caffe2::Caffe2Ops>(m, "Caffe2Ops")
+      .def(py::init([](const std::vector<py::bytes> &ops,
```

Comment:
nit: maybe use std::string as the param type to avoid explicitly casting in your code all over the place?

Reply:
There seems to be some subtle difference between `py::bytes` and `std::string`.